app/models/posthog.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Posthog
+  # Default to the free, hosted Posthog account which we send all dev data
+  URL = ENV.fetch("POSTHOG_URL", "https://app.posthog.com").freeze
+  API_KEY = ENV.fetch("POSTHOG_API_KEY") { "fPfkF4tU2OBcVBZhcshmzC7_VglTw078wuqD3tl01ic" }.freeze
```

Comment:
(p.s. Is this a legit key for a dev instance? Do we all have access to it, if so?)

Reply:
(keeping in mind this is a public repo)

Reply:
Yep, these are public keys (they're exposed in the page source)